var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -47,23 +46,23 @@ def install(self, spec, prefix):
         makefile_inc = []
         makefile_inc.extend([
             'PLAT         = _mac_x',
-            'DSuperLUroot = %s' % self.stage.source_path, #self.stage.path, prefix
+            'DSuperLUroot = %s' % self.stage.source_path,
             'DSUPERLULIB  = $(DSuperLUroot)/lib/libsuperlu_dist.a',
             'BLASDEF      = -DUSE_VENDOR_BLAS',
-            'BLASLIB      = -L%s -llapack %s -lblas' % (spec['lapack'].prefix.lib, spec['blas'].prefix.lib), # FIXME: avoid hardcoding blas/lapack lib names
+            'BLASLIB      = %s %s' % (to_link_flags(spec['lapack'].lapack_shared_lib),to_link_flags(spec['blas'].blas_shared_lib)),
```

Comment:
technically yes, but this does not work a.t.m. for virtual packages. 

If `atlas` does not have it, then someone who use it needs to add it :-)